.github/workflows/build.yaml

```diff
@@ -30,9 +30,8 @@ jobs:
         fetch-depth: 0
  
     - name: Install Nix
-      uses: cachix/install-nix-action@v13
+      uses: cachix/install-nix-action@v16
       with:
-        install_url: https://github.com/numtide/nix-flakes-installer/releases/download/nix-2.4pre20210823_af94b54/install
```

Comment:
I removed this since 2.4 has been out for a while (and now we have 2.5.whatever). Feel free to suggest changes as needed.